charts/sshpiper/templates/deployment.yaml

```diff
@@ -34,8 +34,17 @@ spec:
         ports:
           - containerPort: 2222
         env:
+        {{- if .Values.sshpiper.failtoban.enabled }}
```

Comment:
what about changing cmd instead of env var?
also, this can also apply to image with -full

that way, no change to sshpiper main dockerfile and can keep image smaller

Reply:
> also, this can also apply to image with -full

I was originally thinking this way as well, but it looked to me that failtoban is an essential functionality, which would be used in Kubernetes quite often. So it might be better part of slim image. Helm chart would have to decide (based on `.Values.sshpiper.failtoban.enabled`) if to take full or base.

> what about changing cmd instead of env var?

It just looked to me better to hide the logic and actual plugins paths from the Helm chart and users as much as possible. If these would change in image (`/sshpiperd/plugins/`), they would need to be fixed even in Helm chart and optionally users' configuration. Users would need to fix it anyway if they touch container args on their own, but since there is already configuration interface via PLUGIN which is exactly for selecting plugin, why not to extend it to support more plugins and keep insides hidden.

Of fource, I can adapt it in any way you prefer, it's up to your preference.

I can:
1. leave slim image slim,
2. in Helm chart select default image tag `full-{{ .Chart.AppVersion  }}` for F2B or `{{ .Chart.AppVersion }}` by default
3. in Helm chart instead of PLUGIN, configure container args[] with a list of all enabled plugins; it'll be used for all plugins (even Kubernetes), so that there are not 2 different approaches to deal with selecting plugin(s)
4. in Helm chart optionally expose container args fully to end user to optionally override.

Please comment on what you prefer, I'll adapt. Thanks!

Reply:
thanks please

Reply:
Adapted and force pushed.